css/files.css

```diff
@@ -21,13 +21,13 @@ div[data-cy-files-list].files-list {
 			}	
 		}
 	}
-	.material-design-icon .checkbox-blank-outline-icon {
-		color: var(--ion-color-cool-grey-c5) !important;
+	.material-design-icon.checkbox-blank-outline-icon {
+		color: var(--ion-color-cool-grey-c5);
 	}
 	.material-design-icon.checkbox-marked-icon {
-		color: var(--ion-color-blue-b4) !important;
+		color: var(--ion-color-blue-b4);
 	}
-	.material-design-icon minus-box-icon {
-		color: var(--ion-color-blue-b4) !important;
+	.material-design-icon.minus-box-icon {
+		color: var(--ion-color-blue-b4);
```

Comment:
Dropping the important does not work for me at this icon

Reply:
Feedback re-checked and not confirmed.